src/Components/WebAssembly/WebAssembly.Authentication/src/Interop/AuthenticationService.ts

```diff
@@ -296,7 +302,9 @@ class OidcAuthorizeService implements AuthorizeService {
                 return this.operationCompleted();
             }
         } catch (error) {
-            return this.error(this.getExceptionMessage(error));
+            const message = this.getExceptionMessage(error);
+            this.debug(`Complete sign out error '${message}'`);
+            return this.error(error);
```

Comment:
I believe this should be the message?

```suggestion
            return this.error(message);
```

Reply:
Woops, great catch